crates/runtime/src/func_env.rs

```diff
@@ -19,10 +19,10 @@ pub struct Context {
 ///
 /// SVM is single-threaded.
 /// `Send`, `Sync` and `Clone` are required by `wasmer::WasmerEnv`.
-unsafe impl Send for Context {}
-unsafe impl Sync for Context {}
+unsafe impl Send for FuncEnv {}
+unsafe impl Sync for FuncEnv {}
```

Comment:
These two are a hazard, by the way, but we'll address this in a future PR as it predates this change. SVM is indeed single-threaded, but `wasmer` might pass this between threads and bad stuff will happen...